packages/genesys-spark-components/src/components/stable/gux-pagination-cursor/gux-pagination-cursor.tsx

```diff
@@ -13,7 +13,8 @@ import { buildI18nForComponent, GetI18nValue } from '../../../i18n';
 
 import translationResources from './i18n/en.json';
 import { GuxPaginationCursorDetail } from './gux-pagination-cursor.types';
-import { GuxItemsPerPage } from '../../legacy/gux-pagination-legacy/gux-pagination.types';
+
+import { GuxItemsPerPage } from '../gux-pagination/gux-pagination.types';
```

Comment:
just wanted to confirm ` GuxItemsPerPage` is not the whole `GuxPaginationState` object being exported from this file.

Reply:
No just `GuxItemsPerPage` I am exporting it from `gux-pagination` as the `gux-pagination-cusor` uses `gux-items-per-page` internally and that uses the same export.

Reply:
Ah my bad. I didn't realize until now there's also `GuxItemsPerPage` which is outside of the state object in that file. All good.